cpp/src/io/json/json_tree.cu

```diff
@@ -254,12 +320,485 @@ tree_meta_t get_tree_representation(device_span<PdaTokenT const> tokens,
                                              is_node);
   CUDF_EXPECTS(parent_node_ids_end - parent_node_ids.begin() == num_nodes,
                "parent node id gather mismatch");
+  nvtxRangePop();
+  nvtxRangePop();  // node_parent_ids
   return {std::move(node_categories),
           std::move(parent_node_ids),
           std::move(node_levels),
           std::move(node_range_begin),
           std::move(node_range_end)};
 }
 
+rmm::device_uvector<size_type> hash_node_type_with_field_name(device_span<SymbolT const> d_input,
+                                                              tree_meta_t const& d_tree,
+                                                              rmm::cuda_stream_view stream)
```

Comment:
The function doesn't take a memory resource parameter because it's used internally only?

Reply:
Yes. returned `node_type` will be not be returned from tree traversal.